### PR TITLE
Update dependencies and clean up

### PR DESCRIPTION
**Pull request by mohe2015**
https://github.com/bestguy/sveltestrap/issues/309

Can't update postcss, autoprefixer, webpack to new major versions because storybook breaks then.
Can't update jest, babel-jest to new major versions because of svelte-jester.

### CODE REVIEW

File: src/Tooltip.svelte [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <script>
   import { onDestroy, onMount } from 'svelte';
-  import { createPopper } from '@popperjs/core/dist/esm/popper';
+  import { createPopper } from '@popperjs/core';
```
Comment (bestguy) [inline src/Tooltip.svelte:3]: I'm not sure if we can change this, popper has this annoying dependency on node process and this was causing Sveltestrap to break in the svelte.dev repl

Please see this issue: https://github.com/bestguy/sveltestrap/issues/250

**Review by bestguy — CHANGES_REQUESTED** [review]
Thanks, looks great apart from one comment.

**Comment by mohe2015** [comment]
I'm still working on some more updates. I will undraft this when it's ready.

**Comment by mohe2015** [comment]
@bestguy I think this is ready now. Please thoroughly check that I didn't mess anything up.

**Comment by mohe2015** [comment]
Maybe I or you can do a followup to try to update the remaining packages too.

**Comment by bestguy** [comment]
Thank you for the work! I'll review/merge/release by this weekend.

**Review by bestguy — APPROVED** [review]
LGTM thanks!

**Pull request merged**